unlock-app/src/__tests__/services/walletService.test.js

```diff
@@ -734,6 +734,67 @@ describe('WalletService', () => {
       })
     })
 
+    describe('partialWithdrawFromLock', () => {
+      let lock
+      let account
+
+      beforeEach(() => {
+        lock = '0xd8c88be5e8eb88e38e6ff5ce186d764676012b0b'
+        account = '0xdeadbeef'
+      })
+
+      it('should invoke sendTransaction with the right params', () => {
+        expect.assertions(3)
+        const data = '' // mock abi data for partialWithdraw
+
+        walletService._sendTransaction = jest.fn()
+
+        const ContractClass = class {
```

Comment:
nit: I prefer using something in the name that indicates this is a test helper, either `Mock` or `Test` or whatever makes sense.